frontend/src/stores/CollectionStore.ts

```diff
@@ -67,6 +67,15 @@ export class CollectionStore {
       })
   }
 
+  @action.bound
+  wrapTextWithoutSpaces(text: string): string {
```

Comment:
I would keep this method in the component. It is only used there and is not directly using any values from the store.